awscli/examples/s3/sync.rst

```diff
@@ -68,7 +68,7 @@ Output::
     upload: test2.txt to s3://mybucket/test2.txt
 
 The following ``sync`` command syncs files under a local directory to objects under a specified prefix and bucket by
```

Comment:
Wait, am I reading this right?

It looks like we have a copy/paste error.  We're actually downloading from s3 to local storage so both this sentence (line 70) as well as line 72 should be updated as well.